scripts/install.sh

```diff
@@ -187,6 +187,11 @@ oAuth2ApiEndpoint="undefined"
 patientPickerEndpoint="undefined"
 stage="dev"
 region="us-west-2"
+lambda_latency_threshold=3000
```

Comment:
I was going to suggest you be consistent with your casing, perhaps following camelCaselike the other options, but then I inspected the script further, the casing convention is all over the place anyway.  Nice.  Carry on :)